### PR TITLE
Added Escape(27) for overview

### PR DESCRIPTION
**Pull request by ycktw**
Hi, I'd rather add a short-cut for ESC key to over-view all pages anytime.

### CODE REVIEW

**Comment by bcipolli** [comment]
:+1: for merge. I found this extremely useful, and the code change is tiny.

File: js/impress.js [hunk 1]
```diff
@@ -702,8 +702,11 @@
         //   as another way to moving to next step... And yes, I know that for the sake of
         //   consistency I should add [shift+tab] as opposite action...
         document.addEventListener("keyup", function ( event ) {
-            if ( event.keyCode === 9 || ( event.keyCode >= 32 && event.keyCode <= 34 ) || (event.keyCode >= 37 && event.keyCode <= 40) ) {
+            if ( event.keyCode === 9 || event.keyCode == 27 || ( event.keyCode >= 32 && event.keyCode <= 34 ) || (event.keyCode >= 37 && event.keyCode <= 40) ) {
                 switch( event.keyCode ) {
+		    case 27: // Escape
+			    api.goto( document.getElementById("overview") );
```
Comment (FagnerMartinsBrack) [inline js/impress.js]: This assumes a fixed markup with `id="overview"`. Impress should not dictate how attributes should be named, it should only define behavior for the view (in this case, the HTML).

**Comment by FagnerMartinsBrack** [comment]
:-1: This can be easily achieved with custom code using the public API and a step that acts as a placeholder to simulate an "overview" step.

**Comment by Pierstoval** [comment]
:-1: too, an overview should be set by the user, not by impress itself

**Comment by u982744** [comment]
Down vote. Agree that it's poor design to assume a certain id.

**Comment by bcipolli** [comment]
@ycktw Please use the community feedback to update this pull request. Thanks!

**Comment by FagnerMartinsBrack** [comment]
Closing due to the lack of feedback from the OP. @ycktw, if you want to reopen it, just ping us.

**Pull request closed without merging**